### PR TITLE
Validate front50

### PR DESCRIPTION
**Pull request by lwander**
Closes https://github.com/spinnaker/halyard/issues/106

### CODE REVIEW

**Comment by lwander** [comment]
Forgot to checkin the bumped spinnaker-dependencies version - will fix tomorrow

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/EditPersistentStorageCommand.java [hunk 1]
```diff
@@ -54,6 +56,13 @@
   )
   private String rootFolder = "spinnaker";
 
+  @Parameter(
+      names = "--location",
+      description = "This is only relevant if the bucket you specify doesn't exist yet. In that case, the "
```
Comment (duftler) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/EditPersistentStorageCommand.java]: Is it required if the bucket doesn't exist, or just relevant?

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/EditPersistentStorageCommand.java [hunk 2]
```diff
@@ -63,6 +72,13 @@ protected void executeThis() {
     persistentStorage.setAccountName(isSet(accountName) ? accountName : persistentStorage.getAccountName());
     persistentStorage.setBucket(isSet(bucket) ? bucket : persistentStorage.getBucket());
     persistentStorage.setRootFolder(isSet(rootFolder) ? rootFolder : persistentStorage.getRootFolder());
+    persistentStorage.setLocation(isSet(location) ? location : persistentStorage.getLocation());
+
+    if (persistentStorage.getBucket() == null) {
+      String bucketName = "spin-" + UUID.randomUUID().toString();
+      AnsiUi.raw("Chosen bucket name to be " + bucketName);
```
Comment (duftler) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/EditPersistentStorageCommand.java]: 'Chosen' sounds a bit like the user chose the name. Maybe 'generated'?

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/EditPersistentStorageCommand.java (hunk 1 above)
Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/EditPersistentStorageCommand.java]: Ah, it's required. I'll make that clearer.

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/EditPersistentStorageCommand.java (hunk 2 above)
Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/EditPersistentStorageCommand.java]: Sounds good

**Pull request merged**